README.md

```diff
@@ -1,16 +1,17 @@
 [![CI Status](https://github.com/AY2021S1-CS2103T-F12-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-F12-1/tp/actions)
 
+#HelloFile
+
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+##About
+* HelloFile is a desktop app for managing files, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+Users can tag their file with a nickname to access it easier through the GUI.
+* For the detailed documentation of this project, see the **[HelloFile Product Website](https://ay2021s1-cs2103t-f12-1.github.io/tp/)**.
+
+* This is **a project for CS2103T Software Engineering (SE) class**.<br>
```

Comment:
There is an extra `<br>` at the end, which will increase the gap between this line and the `Acknowledgement`. Is it intended?

Reply:
Didn't realize that. Thanks!